x-pack/plugins/security_solution/public/flyout/entity_details/host_right/index.tsx

```diff
@@ -216,7 +188,10 @@ export const HostPanel = ({
             <FlyoutNavigation
               flyoutIsExpandable={
                 !isPreviewMode &&
-                (isRiskScoreExist || hasMisconfigurationFindings || hasVulnerabilitiesFindings)
+                (isRiskScoreExist ||
+                  hasMisconfigurationFindings ||
+                  hasVulnerabilitiesFindings ||
+                  hasNonClosedAlerts)
```

Comment:
Why are you adding `hasNonClosedAlerts` here? Is it fixing a bug? We should try not to mix changes of functionality in refactor PRs

Reply:
Hi !, yes this is to fix a small bug that I found when refactoring the code

Reply:
I have created a separate PR for this small fix as this PR is too big and risky to be included before FF, but [this](https://github.com/elastic/kibana/pull/200626) fix should be small enough and safe

Reply:
Awesome! I just approved that PR. I'd ship it separately and remove `hasNonClosedAlerts` from here. That way, we'll have a clean refactor-only change-set on this PR